package.json

```diff
@@ -60,9 +60,11 @@
     "body-parser": "^1.18.2",
     "express": "^4.16.2",
     "factory-girl": "^5.0.4",
+    "jsrsasign": "^8.0.12",
     "jwt-simple": "^0.5.1",
     "morgan": "^1.9.1",
     "node-fetch": "^2.6.0",
+    "node-jose": "^1.1.3",
```

Comment:
It's best if you uninstall modules you wont use, to prevent increasing the size of the node_modules folder. One extra module does not seem too much, but they add up eventually.